### PR TITLE
personalisation integration

### PR DESCRIPTION
**Pull request by kdembler**

### CODE REVIEW

File: src/components/InterruptedVideosGallery.tsx [hunk 1]
```diff
@@ -15,12 +15,25 @@ const InterruptedVideosGallery: React.FC<RouteComponentProps> = () => {
 
   const interruptedVideosState = watchedVideos
     .filter((video) => video.__typename === 'INTERRUPTED')
+    // display the newest videos first
+    .reverse()
```
Comment (eldiegod) [inline src/components/InterruptedVideosGallery.tsx:19]: nice

File: src/components/InterruptedVideosGallery.tsx [hunk 2]
```diff
@@ -15,12 +15,25 @@ const InterruptedVideosGallery: React.FC<RouteComponentProps> = () => {
```
Comment (eldiegod) [inline src/components/InterruptedVideosGallery.tsx:15]: Is ContinueWatchingVideos a better name for this? InterruptedVideos sounds a bit odd doesn't it?

Comment (kdembler) [reply src/components/InterruptedVideosGallery.tsx:15]: That's the name we're using internally in other places as well, the `__typename` will have `INTERRUPTED` value. You may be right, but I don't want to do that as part of this PR, it's out of scope tbh

File: src/components/InterruptedVideosGallery.tsx (hunk 1 above)
Comment (kdembler) [reply src/components/InterruptedVideosGallery.tsx:19]: :+1:

File: src/components/InterruptedVideosGallery.tsx (hunk 2 above)
Comment (eldiegod) [reply src/components/InterruptedVideosGallery.tsx:15]: yeah, it's a bit unrelated to the PR

Everything else looks good to me, tho not sure how can this be tested

File: package.json [hunk 3]
```diff
@@ -19,12 +18,9 @@
   },
   "scripts": {
     "start": "react-app-rewired start",
-    "start:prod": "cross-env REACT_APP_QUERY_NODE_URL=https://hydra.joystream.org/graphql REACT_APP_ORION_URL=https://orion.joystream.org/graphql REACT_APP_STORAGE_NODE_URL=https://rome-rpc-endpoint.joystream.org/asset/v0/ yarn start",
     "dev": "yarn start",
     "dev:prod": "yarn start:prod",
```
Comment (drillprop) [inline package.json]: It should be deleted as well.

File: src/shared/__tests__/VideoPreview.test.tsx [hunk 4]
```diff
@@ -1,23 +0,0 @@
...
-          title="Some Video Title"
-          channelName="some channel"
-          posterURL=""
-          views={1000}
-          createdAt={new Date()}
-        />
-      )
-    ).toBeDefined()
-  })
-})
```
Comment (drillprop) [inline src/shared/__tests__/VideoPreview.test.tsx:23]: Shouldn't be this file deleted?

File: src/components/InterruptedVideosGallery.tsx (hunk 2 above)
Comment (kdembler) [reply src/components/InterruptedVideosGallery.tsx:15]: I guess netlify build logs can be useful: https://app.netlify.com/sites/atlas-development/deploys/603ce31945fe680007054178

Also you can checkout the branch locally and change the `REACT_APP_ENV` in `.env` to see how the app behaves

Comment (kdembler) [reply src/components/InterruptedVideosGallery.tsx:15]: for each deploy we will get this in the netlify log:
```
1:52:59 PM: Using STAGING variables
1:52:59 PM: Exposing STAGING_ORION_URL as REACT_APP_ORION_URL=https://orion-staging.joystream.app/graphql
1:52:59 PM: Exposing STAGING_QUERY_NODE_URL as REACT_APP_QUERY_NODE_URL=https://hydra-staging.joystream.app/server/graphql
1:52:59 PM: Exposing STAGING_STORAGE_NODE_URL as REACT_APP_STORAGE_NODE_URL=https://staging-3.joystream.app/storage/asset/v0/
1:52:59 PM: Exposing REACT_APP_ENV=STAGING
```

**Review by drillprop — COMMENTED** [review]
Found two small things

File: package.json (hunk 3 above)
Comment (kdembler) [reply package.json]: Good find, removed

File: src/shared/__tests__/VideoPreview.test.tsx (hunk 4 above)
Comment (kdembler) [reply src/shared/__tests__/VideoPreview.test.tsx:23]: Certainly should be, my fault during rebase, removed now

**Review by mikkio-j — APPROVED** [review]

**Pull request merged**